### PR TITLE
updated the New/Set-AzEnvironment cmdlets to contain azure attestation endpoint

### PR DESCRIPTION
**Pull request by shleiAmy**
## Description
I have submitted a pr to add attestation endpoint to extended endpoints dictionary.
The pr is:   https://github.com/Azure/azure-powershell-common/pull/158
This pr is to update the New/Set-AzEnvironment cmdlets to contain the azure attestation endpoint

## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: tools/Common.Netcore.Dependencies.targets [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <PackageReference Include="Microsoft.Rest.ClientRuntime" Version="2.3.20"/>
     <PackageReference Include="Microsoft.Rest.ClientRuntime.Azure" Version="3.3.19"/>
     <PackageReference Include="Microsoft.Azure.PowerShell.Clients.Aks" Version="1.3.2-preview"/>
-    <PackageReference Include="Microsoft.Azure.PowerShell.Authentication.Abstractions" Version="1.3.2-preview"/>
+    <PackageReference Include="Microsoft.Azure.PowerShell.Authentication.Abstractions" Version="1.3.4-preview"/>
```
Comment (msJinLei) [inline tools/Common.Netcore.Dependencies.targets]: Cannot upgrade refer to Microsoft.Azure.PowerShell.Authentication.Abstractions only

Comment (shleiAmy) [reply tools/Common.Netcore.Dependencies.targets]: could you please explain more details? is there any other version change needed? Thanks.

**Comment by msJinLei** [comment]
@shleiAmy 
- You should add the related test cases for your modification
- change changelog.md
- Your PR replies on upgrading the common lib version in azure powershell. Our team will do it soon.

File: tools/Common.Netcore.Dependencies.targets (hunk 1 above)
Comment (msJinLei) [reply tools/Common.Netcore.Dependencies.targets]: > @msJinLei could you please explain more details? is there any other version change needed? Thanks.

All package references to common lib of 1.3.2 are expected to be upgraded to 1.3.4. not Microsoft.Azure.PowerShell.Authentication.Abstractions only. We will do the upgrading.

Comment (msJinLei) [reply tools/Common.Netcore.Dependencies.targets]: This change is no longer needed. Please rebase you patch with master.

Comment (shleiAmy) [reply tools/Common.Netcore.Dependencies.targets]: I update azEnvironmwnt to add attestation endpoint , why you think the pr is not needed?

Comment (shleiAmy) [reply tools/Common.Netcore.Dependencies.targets]: @msJinLei the rebase is finished.

File: src/Accounts/Accounts/Environment/AddAzureRMEnvironment.cs [hunk 2]
```diff
@@ -316,6 +325,10 @@ public override void ExecuteCmdlet()
                                    nameof(AzureAnalysisServicesEndpointSuffix));
                                 SetEndpointIfBound(newEnvironment, AzureEnvironment.ExtendedEndpoint.AnalysisServicesEndpointResourceId,
                                    nameof(AzureAnalysisServicesEndpointResourceId));
+                                SetEndpointIfBound(newEnvironment, AzureEnvironment.ExtendedEndpoint.AzureAttestationServiceEndpointSuffix,
+                                    nameof(AzureAnalysisServicesEndpointSuffix));
```
Comment (dingmeng-xue) [inline src/Accounts/Accounts/Environment/AddAzureRMEnvironment.cs]: a typo

File: src/Accounts/Accounts/Environment/AddAzureRMEnvironment.cs [hunk 3]
```diff
@@ -316,6 +325,10 @@ public override void ExecuteCmdlet()
...
+                                SetEndpointIfBound(newEnvironment, AzureEnvironment.ExtendedEndpoint.AzureAttestationServiceEndpointResourceId,
+                                    nameof(AzureAnalysisServicesEndpointResourceId));
```
Comment (dingmeng-xue) [inline src/Accounts/Accounts/Environment/AddAzureRMEnvironment.cs]: a typo

**Pull request merged**